_posts/en/newsletters/2024-03-27-newsletter.md

```diff
@@ -0,0 +1,254 @@
+  Core's [feerate estimation][topic fee estimation] using data from a
+  node's local mempool.  Currently, Bitcoin Core generates estimates by
+  recording the block height when each unconfirmed transaction is
+  received, the block height when it is confirmed, and its feerate.
+  When all of that information is known, the delta between received
+  height and confirmed height is used to update an exponentially
+  weighted moving average for a bucket that represents range of
+  feerates.  For example, a transaction that takes 100 blocks to confirm
+  with a feerate of 1.1 sat/vbyte will be incorporated into the average
+  for the 1 sat/vbyte bucket.
+
+  An advantage of this approach is its resistance to manipulation: all
+  transactions must be both relayed (meaning they're available to all
+  miners) and confirmed (meaning they can't violate any consensus
+  rules).  A disadvantage is that it only updates once per block and
+  can lag far behind other estimates that use real-time mempool
+  information.
+
+  Ismail has taken [previous discussion][bitcoin core #27995] about
+  incorporating mempool data into feerate estimates, written some
+  preliminary code, and performed an analysis showing how the current
+  algorithm and a new algorithm compare (not including some safety
+  checks).  A [reply][harding fees] to the thread also linked to
+  [previous research][alm fees] on this topic by Kalle Alm and led to a
+  discussion about whether mempool information should be used to both
+  raiser and lower feerate estimates, or if it should only be used to
+  lower estimates.  The advantage of doing both is that it overall makes
+  the estimates more useful; the advantage of only lowering estimates
+  using mempool data (while only raising estimates using the existing
+  confirmation-based estimation) is that it could be more resistant to
+  manipulation and positive feedback loops.
+
+  Discussion was ongoing as of this writing.
+
+## Selected Q&A from Bitcoin Stack Exchange
+
+*[Bitcoin Stack Exchange][bitcoin.se] is one of the first places Optech
+contributors look for answers to their questions---or when we have a
+few spare moments to help curious or confused users.  In
+this monthly feature, we highlight some of the top-voted questions and
+answers posted since our last update.*
+
+{% comment %}<!-- https://bitcoin.stackexchange.com/search?tab=votes&q=created%3a1m..%20is%3aa
+nswer -->{% endcomment %}
+{% assign bse = "https://bitcoin.stackexchange.com/a/" %}
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [Bitcoin Core 26.1rc2][] is a release candidate for a maintenance release
+  of the network's predominant full node implementation.
+
+- [Bitcoin Core 27.0rc1][] is a release candidate for the next major
```

Comment:
Probably on your radar, but: https://github.com/bitcoin-core/bitcoin-devwiki/wiki/27.0-Release-Candidate-Testing-Guide

(@davidgumberg joining the Recap week)